clippy_lints/src/utils/conf.rs

```diff
@@ -463,6 +463,10 @@ define_Conf! {
     ///
     /// The maximum byte size a `Future` can have, before it triggers the `clippy::large_futures` lint
     (future_size_threshold: u64 = 16 * 1024),
+    /// Lint: UNNECESSARY_BOX_RETURNS,
```

Comment:
```suggestion
    /// Lint: UNNECESSARY_BOX_RETURNS.
```

This list requires a dot at the end, to be parsed correctly. Once you do this, you also have to run `cargo collect-matadata` to update the lint configuration list in our book.